nipy/algorithms/registration/type_check.py

```diff
@@ -0,0 +1,46 @@
+"""
```

Comment:
Maybe this should be a private module?

Consider adding docstrings to public funcs?